next/src/validation/json-logic.ts

```diff
@@ -52,3 +54,54 @@ export function validateJsonLogic(
     return []
   }).flat()
 }
+
+/**
+ * Validates the JSON Logic computed attributes for a given schema.
+ *
+ * @param {SchemaValue} values - Current form values.
+ * @param {NonBooleanJsfSchema} schema - JSON Schema to validate.
+ * @param {ValidationOptions} options - Validation options.
+ * @param {JsonLogicContext | undefined} jsonLogicContext - JSON Logic context.
+ * @param {ValidationErrorPath} path - Current validation error path.
+ */
+export function validateJsonLogicComputedAttributes(
+  values: SchemaValue,
+  schema: NonBooleanJsfSchema,
+  options: ValidationOptions = {},
+  jsonLogicContext: JsonLogicContext | undefined,
+  path: ValidationErrorPath = [],
+): ValidationError[] {
+  const computedAttributes = schema['x-jsf-logic-computedAttrs']
+
+  // if the current schema has no computed attributes, we skip the validation
+  if (!computedAttributes || Object.keys(computedAttributes).length === 0) {
+    return []
+  }
+
+  // Create a copy of the schema
+  const schemaCopy: NonBooleanJsfSchema = { ...schema }
```

Comment:
Probably not an issue in this particular case but just want to highlight that this only creates a shallow copy, so nested objects will still reference the original schema.

Reply:
That's true, I'm aware of that. I did this because at this point I'm hoping we're at the "leaf" schema level, so nested objects would be rare. I can do a deep clone though, it's probably safer.